Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -246,6 +247,27 @@ else
     echo "Skipping adding ardupilot_ws to your home folder."
 fi
 
+
+if maybe_prompt_user "Add ardupilot_gazebo to your home folder [N/y]?" ; then
+    if [ ! -d $AP_GZ_ROOT ]; then
+        sudo apt install libgz-sim7-dev rapidjson-dev
+        git clone https://github.com/ArduPilot/ardupilot_gazebo
+        pushd $AP_GZ_ROOT
+        mkdir build && pushd build
```

Comment:
Need to create the workspace directory first, then clone, then cd into the project, then make build dir and build.

```bash
mkdir -p $AP_GZ_ROOT/src
pushd $AP_GZ_ROOT/src
git clone https://github.com/ArduPilot/ardupilot_gazebo
pushd ardupilot_gazebo
mkdir build && pushd build
```